omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
it still doesn't load the tags the first time. Maybe the loadTags() function runs before $scope.thisPersonUuid gets initialised in the init method